server/database/config/build.sql

```diff
@@ -1,21 +1,34 @@
 BEGIN;
 
-DROP TABLE IF EXISTS users, exercises, food, food_category, food_type, nutrition, UserExerciseRelation, UserFoodRelation, news CASCADE;
+DROP TABLE IF EXISTS users, activity, exercises, food, food_category, food_type, nutrition, UserExerciseRelation, UserFoodRelation, news CASCADE;
+
+
+CREATE TABLE activity(
+    id SERIAL PRIMARY KEY,
+    activity_name VARCHAR(100),
+    activity_value Float
+);
+
 
 CREATE TABLE users(
 id SERIAL PRIMARY KEY,
-first_name VARCHAR(100) NOT NULL,
-last_name VARCHAR(100) NOT NULL,
-email VARCHAR (100) NOT NULL UNIQUE,
-password VARCHAR(250) NOT NULL ,
-gender CHAR(1),
-height INTEGER,
-weight INTEGER,
+lastName VARCHAR(100),
+email VARCHAR (100),
+password VARCHAR(250),
+firstName VARCHAR(100) NOT NULL,
+gender VARCHAR(1),
 age INTEGER,
-image TEXT,
-daily_calories_goal INTEGER
+weight INTEGER,
+height INTEGER,
+goalWeight INTEGER,
+dailyCaloriesGoal INTEGER,
```

Comment:
Change data type in height and daily calorie goal to float, when you insert float data to an integer type feild, it'll be automatically estimated to the highest integer, therefore incorrect data, that's why we need float data type (keep in mind height is in meters so it has to ba float)